doc/panes/editor.rst

```diff
@@ -2,21 +2,30 @@
 Editor
 ######
 
-Spyder's multi-language **Editor** integrates a number of powerful tools right out of the box for an easy to use, efficient editing experience.
-The Editor's key features include syntax highlighting (``pygments``); real-time code and style analysis (``pyflakes`` and ``pycodestyle``); on-demand completion, calltips and go-to-definition features (``rope`` and ``jedi``); a function/class browser, horizontal and vertical splitting, and much more.
+Spyder's multi-language **Editor** pane is the key element of the IDE, where you can create, open, and modify source files.
+The Editor offers a variety of core features, such as autocompletion, real-time analysis, syntax highlighting, horizontal and vertical splitting, and much more.
+In addition, it integrates a number of powerful tools for an easy to use, efficient editing experience.
 
 .. image:: /images/editor/editor-standard.png
    :alt: Spyder's Editor panel, split horizontally and with style analysis
 
-:guilabel:`Outline Explorer` (function/class/method browser) and horizontal/vertical splitting capabilities:
+==============
+Key components
+==============
 
-.. image:: /images/editor/editor-outline-standard.png
-   :alt: Spyder outline panel, showing the functions/classes/methods in a file
+The Editor pane consists of the following areas:
 
-Real-time code and style analysis with ``pyflakes`` and ``pycodestyle``:
+.. image:: /images/editor/editor-components.png
+   :alt: Spyder's Editor pane, showing its different areas (described below)
 
-.. image:: /images/editor/editor-inset-code-analysis.png
-   :alt: A snippit of code in the Spyder Editor, showing code style warnings
+1. The left sidebar shows line numbers and displays any code analysis errors/warnings that exist in the current file.
+   Clicking a line number selects the code/text on that line.
```

Comment:
What about this?

```suggestion
1. The left sidebar shows line numbers and displays any code analysis warnings that exist in the current file.
   Left-clicking a line number selects the text on that line, and right-clicking to the right of it sets a :ref:`breakpoint <debugging-breakpoints>`.
```

You'll also need to add an appropriately named reference target in `panes/debugging.rst` like so (FYI, I really should go in and add ref targets to all headings, so we don't need to do this in the future):

```rst
.. image:: /images/debugging/debugging-multiline.gif
   :alt: Debugger showing multiline debugger



.. _debugging-breakpoints:

===========
Breakpoints
===========
```

I also simplified it a touch elsewhere to compensate.

Reply:
Isn't it **left**-clicking to the right of the line number to set a breakpoint?

Reply:
You're totally right @jitseniesen! Thanks for catching that.

Reply:
My suggestion

```suggestion
1. The left sidebar shows line numbers and displays any code analysis errors/warnings that exist in the current file.
   Left-clicking a line number selects the text on that line, and doing it to the right of it sets a :ref:`breakpoint <debugging-breakpoints>`.
```

My suggestion to address @jitseniesen's comment.

Reply:
Thanks, I should have noticed that too, sorry. I've edited the original, and repeat it here for clarity:

```suggestion
1. The left sidebar shows line numbers and displays any code analysis warnings that exist in the current file.
   Clicking a line number selects the text on that line, and clicking to the right of it sets a :ref:`breakpoint <debugging-breakpoints>`.
```

Reply:
Ok, I agree with this suggestion.

Reply:
Just so it doesn't get lost, @hyounes4560 , in addition to the suggestion, you'll also need to add the indicated ref target label to `panes/debugging.rst` as [shown above](https://github.com/spyder-ide/spyder-docs/pull/322#discussion_r944763545). Thanks!